lib/_http_outgoing.js

```diff
@@ -408,7 +407,7 @@ OutgoingMessage.prototype.write = function(chunk, encoding, callback) {
     return true;
   }
 
-  if (!util.isString(chunk) && !util.isBuffer(chunk)) {
+  if (!(typeof chunk === 'string' || util.isBuffer(chunk))) {
```

Comment:
You might as well change `util.isBuffer(chunk)` to `chunk instanceof Buffer` while you're here.